sprimber-engine/src/main/java/com/griddynamics/qa/sprimber/engine/executor/ErrorMapper.java

```diff
@@ -50,6 +52,12 @@ public ErrorMapper(List<String> skippedExceptionNames,
         this.pendingExceptionAnnotations = pendingExceptionAnnotations;
     }
 
+    @PostConstruct
+    private void sortCollections() {
+        Collections.sort(skippedExceptionNames);
```

Comment:
As I see - you always convert this two lists to array (in method parseThrowable)
I think it's better to do it only one time and store arrays instead of lists.
Also I don't understand why did you decide to use @PostConstruct if you can write this logic inside constructor